packages/sveltekit/package.json

```diff
@@ -13,6 +13,10 @@
   "module": "build/esm/index.server.js",
   "browser": "build/esm/index.client.js",
   "types": "build/types/index.types.d.ts",
+  "exports": {
+    "browser": "./build/esm/index.client.js",
+    "node": "./build/cjs/index.server.js"
+  },
```

Comment:
Wondering if we should have esm and cjs for both browser and node... (by adding objects with `import` and `require` fields)

https://nodejs.org/api/packages.html#conditional-exports:~:text=%22import%22%3A%20%22./feature%2Dnode.mjs%22%2C%0A%20%20%20%20%20%20%22require%22%3A%20%22./feature%2Dnode.cjs%22%0A%20%20%20%20%7D%2C

Reply:
we should.

Ideally exports fields look something like so:

<img width="320" alt="image" src="https://github.com/getsentry/sentry-javascript/assets/18689448/1fda899a-ea95-48b6-8d7d-558d650eb6e0">

(from https://abhiprasad.github.io/talks/2023/publishing-javascript-libraries-made-easy.pdf)

I don't think we can get the types separated, so we can ignore that field.

Reply:
Actually esm is broken because of our http integration, so maybe we just dont expose that for now.

Reply:
Shameless plug @AbhiPrasad :P

Reply:
@AbhiPrasad what's the advantage of the `package.json`  export?

Reply:
@AbhiPrasad so we want this (considering server-side ESM is broken), correct?
![image](https://github.com/getsentry/sentry-javascript/assets/8420481/cfeba5c0-7de4-4ebe-b99f-8c42f02b4f0d)

Reply:
Tested the version from the screenshot and it still seems to work so going with it.